src/middleware/nunjucksLoader.middleware.ts

```diff
@@ -17,8 +17,9 @@ export default class NunjucksLoader {
   public constructor(
     @inject(TYPES.CDN_HOST) private CDN_HOST: string,
     @inject(TYPES.PIWIK_URL) private PIWIK_URL: Optional<string>,
-    @inject(TYPES.PIWIK_SITE_ID) private PIWIK_SITE_ID: Optional<string>) {
-  }
+    @inject(TYPES.PIWIK_SITE_ID) private PIWIK_SITE_ID: Optional<string>,
+    @inject(TYPES.PIWIK_LANDING_PAGE_START_GOAL_ID) private PIWIK_LANDING_PAGE_START_GOAL_ID: Optional<string>
```

Comment:
These shouldn't optional. Here's another thought, rather than injecting piwik config separately, should we created a PiwikConfig interface and inject it here instead? Similar to AuthConfig.